src/views/ont/ONTPoolIndex.vue

```diff
@@ -168,6 +185,19 @@ export default {
     async fetchPools() {
       return await this.fetchWithQueryParams(this.fetchOntPools, this.filterOptions)
     },
+    async updateLocations(locationsData) {
+      this.updatedPools = this.updatedPools.map((pool) => {
```

Comment:
Again it looks like you are formatting the locations in a particular way so maybe this method belongs as a helper.